.github/workflows/docs.yaml

```diff
@@ -0,0 +1,80 @@
+name: Covasim Docs workflow
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    if: github.repository == 'InstituteforDiseaseModeling/covasim'
+    strategy:
+      fail-fast: false
+      max-parallel: 8
+      matrix:
+        python-version: ['3.7']
+    name: Install and Generate Docs
+    steps:
+      - name: Set repo owner env variable
+        shell: python
+        run: print("::set-env name=GITHUB_OWNER::{}".format('${{github.repository}}'.split('/')[0]))
+      - name: Set repo owner env variable
+        shell: python
+        run: print("::set-env name=GITHUB_OWNER::{}".format('${{github.repository}}'.split('/')[0]))
+      - name: Set PYTHONPATH
+        run: |
+          mkdir -p $HOME/.cache/site-packages
+          echo "::set-env name=PYTHONPATH::$HOME/.cache/site-packages"
+      - name: Checkout Docs Repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ env.GITHUB_OWNER }}/covasim-docs
+          path: covasim-docs
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        with:
+          path: covasim
+      - uses: actions/setup-python@master
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      - name: Cache Packages
+        id: cache-packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/ # This path is specific to Ubuntu
+          key: docs-packages-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('requirements.txt') }}--${{ hashFiles('docs/requirements.txt') }} # Look to see if there is a cache hit for the corresponding requirements file
+          restore-keys: |
+            docs-packages-${{ runner.os }}-${{ matrix.python-version }}- # Pip install first will make setup.py much faster and the cache will make pip install fast
+      - name: Install Covasim
+        run: |
+          pushd covasim
+          python setup.py develop --install-dir ~/.cache/site-packages
+          pip3 install -r docs/requirements.txt
+          popd
+      - name: Build Docs
+        run: |
+          pushd covasim/docs
+          make html
+          popd
+
+          rm -rf covasim-docs/*
```

Comment:
I know there are flags on rsync to help with deleting missing files, but I figure it's probably easier to just start clean every time

Reply:
this looks terrifying but it's on the server so i guess it's ok :)

Reply:
heh, very true. I immediately rsync afterwards. We aren't force pushing, and worse comes to worse we can revert!